go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/rantav/go-archetype
+module github.com/diegosz/go-archetype
```

Comment:
Hi @diegosz,

Thank you for your contribution. I noticed that you renamed the module to point to your repository. Could you please revert this change?

Reply:
oops, sorry about that

Reply:
Please close this PR, already made this one: https://github.com/rantav/go-archetype/pull/29